common/helpers.go

```diff
@@ -0,0 +1,38 @@
+package common
+
+import (
+	"context"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
+	"google.golang.org/grpc/metadata"
+)
+
+func GetFields(ctx context.Context) logging.Fields {
+	headers := GetMetadata(ctx)
+	requestID := headers[RequestIDMetadataKey]
+	// Remove the main request ID from headers map since it's logged separately
+	delete(headers, RequestIDMetadataKey)
+	return logging.Fields{
+		RequestIDLogKey, requestID,
```

Comment:
Can we unify the two? That is, treat requestID the same as another header?

Reply:
Yes we can unify. Reason I kept it separate is because we currently have the requestid in its own column in our kusto tables. Made the change to unify